### DISCUSSION TITLE
Is it possible to register more than one formula pointing to the same package?

### DISCUSSION THREAD

**Discussion by andreaangiolillo** [Homebrew/discussions#3004] [Writing Formulae/Casks] [answered]
Hello,

I would like to know if it is possible to register more than one formula pointing to the same package.

**Example:**
The idea is that `brew install tool-1` and `brew install db-tool-1` would install the same binary.

Thanks!

**Comment by SMillerDev** [comment, marked as answer]
That's what we have the `Aliases` directory for. You can see them in action in Homebrew/core.
